src/blocks/custom.rs

```diff
@@ -147,12 +147,8 @@ async fn update_bar(
 
 pub async fn run(config: toml::Value, mut api: CommonApi) -> Result<()> {
     let config = CustomConfig::deserialize(config).config_error()?;
-    let format_config = FormatConfig {
-        full: config.format.full,
-        short: config.format.short.or(Some(" $short_text |".parse()?))
-    };
     let mut widget = api.new_widget().with_format(
-        format_config.with_default("{ $icon|} $text ")?
+        config.format.with_defaults("{ $icon|} $text ", "{ $icon|}{ $short_text|} ")?
```

Comment:
Event if `$short_text` is not available, this string will evaluate to something non-empty, thus the block will be hide-able. So something like this is more correct: `"{ $icon|} $short_text `.